rc/extra/autorestore.kak

```diff
@@ -14,6 +14,9 @@ def autorestore-restore-buffer -docstring "Restore the backup for the current fi
 
         if [ -z "${backup_path}" ]; then
             exit
+        elif [ ${kak_buffile} -nt ${backup_path} ]; then
```

Comment:
Unfortunately, `-nt` is not posix... IIRC you need to use `find ... -newer $kak_buffile` to do the same.

This also reminds me why we were using `backup_path=$(ls -1t ."${kak_bufname}".kak.* 2>/dev/null | head -n 1)` to get the candidate backup buffer. So we'd need to fix that as well (get the most recent backup file).